### PR TITLE
Output simple run config object with example notebook

### PR DESCRIPTION
**Pull request by jeswan**
This PR outputs a `simple_run_config.json` object when using the `jiant` simple API. `simple_run_config.json`can be loaded to repeat a `jiant` experiment. This is demonstrated in the example notebook in this PR.

### CODE REVIEW

File: jiant/proj/main/runscript.py [hunk 1]
```diff
@@ -211,6 +211,7 @@ def run_loop(args: RunConfiguration, checkpoint=None):
     ):
         os.remove(os.path.join(args.output_dir, "checkpoint.p"))
 
+    py_io.write_file(args.to_json(), os.path.join(args.output_dir, "run_config.json"))
```
Comment (jeswan) [inline jiant/proj/main/runscript.py:214]:
```suggestion
```

File: examples/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## Example Notebooks
 
+* **Fine-tuning on a single task with the simple API** [[Notebook](./notebooks/simple_api_fine_tuning.ipynb)] [[Colab](https://colab.research.google.com/github/jiant-dev/jiant/blob/master/examples/notebooks/simple_api_fine_tuning.ipynb.ipynb)]
```
Comment (zphang) [inline examples/README.md:5]: Mind updating this (and the other bullet points) with the tasks/model names?

File: examples/notebooks/simple_api_fine_tuning.ipynb [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rY-7weGtIEUX",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "%%capture\n",
+        "!git clone https://github.com/jiant-dev/jiant.git\n",
```
Comment (zphang) [inline examples/notebooks/simple_api_fine_tuning.ipynb:48]: Checkout specific hash?

File: examples/README.md (hunk 2 above)
Comment (zphang) [reply examples/README.md:5]: On second thought, I don't feel super strongly about this. Could go either way.

File: examples/notebooks/simple_api_fine_tuning.ipynb [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+        "colab": {}
+      },
+      "source": [
+        "args = run.RunConfiguration(\n",
+        "    run_name=RUN_NAME,\n",
+        "    exp_dir=EXP_DIR,\n",
+        "    data_dir=DATA_DIR,\n",
+        "    model_type=MODEL_TYPE,\n",
+        "    tasks=TASK_NAME,\n",
+        "    train_examples_cap=1024,\n",
```
Comment (zphang) [inline examples/notebooks/simple_api_fine_tuning.ipynb:202]: Do we need this option?

Comment (jeswan) [inline examples/notebooks/simple_api_fine_tuning.ipynb:202]:
```suggestion
```

Comment (jeswan) [reply examples/notebooks/simple_api_fine_tuning.ipynb]: Removed

File: examples/README.md (hunk 2 above)
Comment (jeswan) [inline examples/README.md:5]:
```suggestion
* **Fine-tuning on a single task with the simple API** [[Notebook](./notebooks/simple_api_fine_tuning.ipynb)] [[Colab](https://colab.research.google.com/github/jiant-dev/jiant/blob/master/examples/notebooks/simple_api_fine_tuning.ipynb)]
```

File: examples/notebooks/simple_api_fine_tuning.ipynb [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rY-7weGtIEUX",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "%%capture\n",
+        "!git clone https://github.com/jiant-dev/jiant.git\n",
```
Comment (jeswan) [inline examples/notebooks/simple_api_fine_tuning.ipynb:48]:
```suggestion
        "!git clone https://github.com/jiant-dev/jiant.git\n",
        "%cd jiant\n",
        "!git checkout 572441964e859d00fa2071c0888dd4ab6a8d3619\n",
        "%cd ..\n"
```

File: examples/notebooks/simple_api_fine_tuning.ipynb (hunk 3 above)
Comment (jeswan) [reply examples/notebooks/simple_api_fine_tuning.ipynb:48]: done

**Review by zphang — APPROVED** [review]

**Pull request merged**